CHARTER.md

```diff
@@ -0,0 +1,138 @@
+
+The mailing list at groups.google.com/forum/#!forum/okd-wg will be used as a place to call for and publish group decisions, and to hold discussions in general.
+
+### Working Group Membership
+All active members of the Working Group are listed in the `MEMBERS.md` file with their name.
+
+New members can apply for membership by creating an Issue or Pull Request on the repository on GitHub indicating their desire to join.
+
+Membership can be surrendered by creating an Issue stating this desire, or by creating a Pull Request to remove the own name from the members list.
+
+### Decision Process
+This group will seek consensus decisions. After public discussion and consideration of different opinions, the Chair and/or Co-Chair will record a decision and summarize any objections. 
+
+All WG members who have joined the GitHub group at least 21 days prior to the vote are eligible to vote. This is to prevent people from rallying outside supporters for their desired outcome. 
+
+When the group comes to a decision in a meeting, the decision is tentative. Any group participant may object to a decision reached at a meeting within 7 days of publication of the decision on the GitHub Issue and/or mailing list. That decision must then be confirmed on the GitHub Issue via a Call for Agreement.
+
+The Call for Agreement, when a decision is required, will be posted as a GitHub Issue or Pull Request and must be announced on the mailing list. It is an instrument to reach a time-bounded lazy consensus approval and requires a voting period of no less than 7 days to be defined (including a specific date and time in UTC).
+
+Each Call for Agreement will be considered independently, except for elections of Chairs.
+
+The Chairs will submit all Calls for Agreement unless vague, unprofessional, off-topic or lacking sufficient detail to determine what is being agreed. 
+
+In the event that a Call for Agreement falls under the delegated authority or within a chartered Sub-Working Group, the Call for Agreement must be passed through the Sub-Working Group before receiving Working Group consideration. 
+
+A Call for Agreement may require quorum of Chairs under the circumstances outlined in the Charter and Governing Documents section.
+
+A Call for Agreement is mandatory when:
+* A Chair determines that the topic requires a Call for Agreement.
+* When petitioned by members of the Working Group and submitted to the Chairs to call a vote unless rejected for cause. 
+* Technical decisions that add, remove, or change dependencies and requirements.
+* Revoke a previous decision made by the Call for Agreement process.
+* Approval of a Sub-Working Group when such SWG has any delegated authority.
+
+Once the Call for Agreement voting period has elapsed, all votes are counted, with at least a 51% majority needed for consensus. A Chair will then declare the agreement “accepted” or “declined”, ending the Call for Agreement. 
```

Comment:
Further clarify that 51% of votes need agree, not 51% of working group since no quorum requirement.
```suggestion
Once the Call for Agreement voting period has elapsed, all votes are counted, with at least a 51% majority of votes needed for consensus. A Chair will then declare the agreement “accepted” or “declined”, ending the Call for Agreement. 
```